receiver/libhoneyreceiver/internal/parser/parser.go

```diff
@@ -81,8 +82,17 @@ func ToPdata(dataset string, lhes []libhoneyevent.LibhoneyEvent, cfg libhoneyeve
 			ls.Scope().SetName(ss.LibraryName)
 			ls.Scope().SetVersion(ss.LibraryVersion)
 			foundScopes.Scope[scopeName].ScopeLogs.MoveAndAppendTo(ls.LogRecords())
+
+			tr := resultTraces.ResourceSpans().AppendEmpty()
```

Comment:
should the code here call `ToPTraceSpan` at some point?

Reply:
Yep. Should have marked it as a draft. My bad.